PurchasesCoreSwift/Public/PurchaseOwnershipType.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PurchaseOwnershipType.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 6/24/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCPurchaseOwnershipType) public enum PurchaseOwnershipType: Int {
```

Comment:
why do we need the RC prefix for the objc version and not swift?

Reply:
because obj-c is dumb. 
There's no concept of namespacing in objective-c, so it's convention to use a prefix as a way to prevent class names from colliding (i.e.: if an app had its own class called `PurchaseOwnershipType` and it also imported our SDK and our class didn't have the prefix, which one is used is undefined. 

Conversely, if someone's app is using `RC` as the prefix, they'd need to use something else instead to use our SDK. 

It's an old language, after all. 

In Swift, things are namespaced by module, in our case, `PurchasesCoreSwift`, so this gets disambiguated as `PurchasesCoreSwift.PurchaseOwnershipType` vs `MyApp.PurchaseOwnershipType`. 

As for the `RC` prefix, most companies use a shorthand of their company name. Some iOS framework use the framework name, i.e.: `SK`, since they have multiple classes with similar name (like `Transaction`) for different frameworks. 

Classes from `Foundation` (imported on line 9) use the prefix `NS`, which, believe it or not, comes from the NextStep days.

Reply:
Swift has namespaces 🎉 ObjC doesn't, so you need *something* to namespace it and people in ObjC land settled on the 2-letter convention.
check this out: https://nshipster.com/namespacing/

Reply:
thank you @aboedo :)

Reply:
Ahhh I was too slow.